volttron/platform/vip/agent/core.py

```diff
@@ -1008,6 +1007,7 @@ def connect_callback():
                             binding['routing_key'] == router_key:
                         router_connected = True
                         break
+            router_connected = True
```

Comment:
Earlier we set router_connected=True only if there are bindings.  but it is outside of "if bindings:" so wont the flag get set even if there was attribute error earlier?